spec/javascripts/govuk_publishing_components/analytics-ga4/gtm-link-click-tracker.spec.js

```diff
@@ -0,0 +1,416 @@
+/* eslint-env jasmine, jquery */
+
+describe('GOVUK.analyticsGA4.linkClickTracker', function () {
+  'use strict'
+  var GOVUK = window.GOVUK
+  var $ = window.jQuery
```

Comment:
This might be a pain, but can we avoid using jQuery? We're going to have to eventually remove all jQuery from the site, even tests, so the frontend team agreed a while ago not to add any more.

Reply:
I suspect one thing that will be tricky is removing the click event listeners, but there's ways around this if it's a problem, let me know.